manager-rs/dauth-service/src/tasks/update_users.rs

```diff
@@ -137,37 +164,37 @@ async fn handle_user_update(context: Arc<DauthContext>, user_id: &str) -> Result
             )
             .await?;
         }
-        transaction.commit().await?;
+        for (xres_star_hash, _, rand) in shares {
+            database::key_share_state::add(
+                &mut transaction,
+                xres_star_hash,
+                backup_network_id,
+                user_id,
+                rand,
+            )
+            .await?;
+        }
+
+        // drop rand before sending
+        // TODO: allow backups to have rand? Would allow them to check res
```

Comment:
Backups should be able to check res, so we should add the rand to the appropriate messages. That was my oversight!

Reply:
(not essential to address in this PR)